spring-boot/current-redhat/cache/booster.yaml

```diff
@@ -2,13 +2,13 @@ metadata:
   app:
     launcher:
       runsOn:
-        - '!starter'
+      - '!starter'
 name: Spring Boot - Cache Example
 description: Booster to demonstrate how to use a cache server Spring Boot on OpenShift.
 source:
   git:
     url: https://github.com/snowdrop/spring-boot-cache-booster
-    ref: redhat
```

Comment:
Is it correct to update untagged boosters?

Reply:
I'm actually not sure why that booster was done that way. It was the only one using the `environments` section. Switching from branch head to tag is better, in my opinion.

Reply:
I also don't understand this change

Reply:
Does the change in `master` perhaps fix the issue?

Reply:
I noticed that this PR conflicts with master on this file

Reply:
Ah yes, this PR depends on the change in master